CHANGES.md

```diff
@@ -3,6 +3,9 @@
 - Fix a bug making all loky workers crash on Windows for Python>3.7 when using
   a virtual environment (#216).
 
+- Copy the environment variables in the child process for ``LokyProcess``.
+  This allow setting env variables before loading any module. (#217)
```

Comment:
This should be updated to reflect the fact that it's now also possible to pass specific environment variables just for the worker processes without changing the environment of the parent process.

Reply:
Maybe also add a disclaimer that this feature is does not work with Python 2.7 and Windows.